src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/CondEquipCompDispensable.java

```diff
@@ -0,0 +1,74 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Name("Equippable Component - Can Be Dispensed")
+@Description("Whether an item can be dispensed by a dispenser. "
+	+ "NOTE: Equippable component elements are experimental. Thus, they are subject to change and may not work as intended.")
+@Example("""
+	if {_item} can be dispensed:
+		add "Dispensable" to lore of {_item}
+	""")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	if {_component} is not able to be dispensed:
+		allow {_component} to be dispensed
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class CondEquipCompDispensable extends PropertyCondition<EquippableWrapper> implements EquippableExperiment {
+
+	static {
+		List<String> patterns = new ArrayList<>(Arrays.asList(getPatterns(PropertyType.CAN, "be dispensed", "equippablecomponents")));
+		patterns.addAll(Arrays.asList(getPatterns(PropertyType.BE, "(able to be dispensed|dispensable)", "equippablecomponents")));
+
+		Skript.registerCondition(CondEquipCompDispensable.class, ConditionType.PROPERTY, patterns.toArray(String[]::new));
+	}
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		//noinspection unchecked
+		setExpr((Expression<? extends EquippableWrapper>) exprs[0]);
+		setNegated(matchedPattern % 2 == 1);
+		return true;
+	}
+
+	@Override
+	public boolean check(EquippableWrapper wrapper) {
+		return wrapper.getComponent().isDispensable();
+	}
+
+	@Override
+	protected String getPropertyName() {
+		return "dispensable";
+	}
+
+	@Override
+	public String toString(@Nullable Event event, boolean debug) {
```

Comment:
Maybe you could use the static toString method? https://github.com/SkriptLang/Skript/blob/0b0b3da02e61ebb8019bb9d0922aaa8741731759/src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java#L223